src/data/bluetooth.ts

```diff
@@ -84,7 +91,24 @@ export const subscribeBluetoothAdvertisements = (
     "_bluetoothDeviceRows",
     () => Promise.resolve<BluetoothDeviceData[]>([]), // empty array as initial state
 
-    subscribeUpdates,
+    subscribeBluetoothAdvertisementsUpdates,
     conn,
     onChange
   );
+
+export const subscribeBluetoothConnectionAllocations = (
+  conn: Connection,
+  onChange: (bluetoothAllocationsData: BluetoothAllocationsData[]) => void,
+  configEntryId?: string
+): Promise<() => Promise<void>> => {
+  const params: any = {
```

Comment:
```suggestion
  const params: MessageBase = {
```
You can use this type here

Reply:
a5e14395dc3c0aada06c01399faffc8ff26a9519